samples/test/clean.js

```diff
@@ -0,0 +1,249 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This module contains utility functions for removing unneeded, stale, or
+ * orphaned resources from test project.
+ *
+ * Removes:
+ * - Datasets
+ * - Training pipelines
+ * - Models
+ * - Endpoints
+ * - Batch prediction jobs
+ */
+const MAXIMUM_AGE = 3600000 * 24 * 2; // 2 days in milliseconds
+const MAXIMUM_NUMBER_OF_DELETIONS = 10; // Avoid hitting LRO quota
+const TEMP_RESOURCE_PREFIX = 'temp';
+const LOCATION = 'us-central1';
+
+// All AI Platform resources need to specify a hostname.
+const clientOptions = {
+  apiEndpoint: 'us-central1-aiplatform.googleapis.com',
```

Comment:
I'd make this a template string that references `${LOCATION}`, so that we don't set `us-central1` in several places.

Reply:
Acknowledged.

Unfortunately, the GCE region (LOCATION) does not always line up 1-to-1 with the hostname.